src/Umbraco.AuthorizedServices/Services/IAuthorizedServiceCaller.cs

```diff
@@ -57,4 +57,11 @@ public interface IAuthorizedServiceCaller
 
     Task<string> SendRequestRawAsync<TRequest>(string serviceAlias, string path, HttpMethod httpMethod, TRequest? requestContent = null)
         where TRequest : class;
+
+    /// <summary>
+    /// Sends a request to retrieve the API key for a service.
+    /// </summary>
+    /// <param name="serviceAlias"></param>
+    /// <returns></returns>
+    Task<string> GetApiKeyAsync(string serviceAlias);
```

Comment:
As per my update in the other API, think you can change this method signature to `string? GetApiKey(string serviceAlias)` and treat it synchronously.

Reply:
Updated